provider/cmd/helpers.go

```diff
@@ -22,10 +24,6 @@ func addCmdFlags(cmd *cobra.Command) {
 		panic(err.Error())
 	}
 
-	if err := cmd.MarkFlagRequired(flags.FlagHome); err != nil {
```

Comment:
don't we still want this?

Reply:
if it marked as required then it prompts to set `--home` and ignores default value from above